recipes/libusrsctp/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibusrsctpConan(ConanFile):
+    name = "libusrsctp"
```

Comment:
libusrsctp, usrsctp or usrsctplib?

Reply:
I am for the libusrsctp although upstream does not ever mention this name.

Reply:
Follow same project name, `usrsctp`. VCPKG also uses usrsctp

Reply:
Ok, let's switch